cms/standard_pages/models.py

```diff
@@ -38,3 +40,88 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class IndexPage(BasePage):  # type: ignore[django-manager-missing]
+    template = "templates/pages/index_page.html"
+
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage"]
+    subpage_types: ClassVar[list[str]] = ["IndexPage", "InformationPage"]
+
+    summary = models.TextField()
+    featured_items = StreamField(
+        [("featured_item", RelatedContentBlock())],
+        help_text="Leave blank to automatically populate with child pages. Only published pages will be displayed.",
```

Comment:
Our rule is to mark any user-displayed text as translatable.
so `from django.utils.translation import gettext_lazy as _` and `help_text=_("Leave blank...")`